.linterconfig.yaml

```diff
@@ -0,0 +1,6 @@
+# Turn the components of the linter individualy on / off.
+clangformat: on
+cpplint: on
+yapf: on
+pylint: on
+block-commits: off
```

Comment:
this is a completely new file. I think these things should go into the example linterconfig (which you did already) and we should not push an actual linterconfig here. It can be set up later as part of the install procedure within the repository that is using it. But I am happy to discuss.